src/main/java/lotto/service/LottoService.java

```diff
@@ -0,0 +1,48 @@
+package lotto.service;
+
+import lotto.utility.LottoWinningStaticsCalculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class LottoService {
+    private final static int lottoPrice = 1000;
```

Comment:
상수 선언 시 키워드는 `static final` 순으로 작성해주시고, 이름은 관례에 따라 대문자로 작성해보시면 좋을 것 같습니다.
```suggestion
    private static final int LOTTO_PRICE = 1000;
```